src/main/java/com/fundacionjala/movies/Movie.java

```diff
@@ -0,0 +1,54 @@
+package com.fundacionjala.movies;
+
+public abstract class Movie {
+
+    protected final String title;
```

Comment:
How you mentioned me, this variable is final and remove the "Code" variable because was unnecessary.